### ISSUE TITLE
Is this the upstream github for unrar?

### ISSUE THREAD

**Issue by fennectech** [pmachapman/unrar#3]

**Comment by pmachapman** [comment]
No, this is just a mirror of the unrar source code from rarlab.com. I took all of the unrar source zip files on this website, and created a commit for each of them, in version order.

**Comment by fennectech** [comment]
I wound up finding the problem anyways.   Was a zip file pretending to be a rar file (had a wrong file extension) xD.
